pkg/apis/serving/fieldmask.go

```diff
@@ -136,7 +137,7 @@ func KeyToPathMask(in *corev1.KeyToPath) *corev1.KeyToPath {
 // PodSpecMask performs a _shallow_ copy of the Kubernetes PodSpec object to a new
 // Kubernetes PodSpec object bringing over only the fields allowed in the Knative API. This
 // does not validate the contents or the bounds of the provided fields.
-func PodSpecMask(in *corev1.PodSpec) *corev1.PodSpec {
+func PodSpecMask(in *corev1.PodSpec, features *config.Features) *corev1.PodSpec {
```

Comment:
How hard would it be to plumb context into here instead?

Reply:
That should be easy enough. I'll give it a go.